src/BlockchainReader/Bitcoin.test.ts

```diff
@@ -0,0 +1,110 @@
+import { PoetTimestamp } from '@po.et/poet-js'
+import { allPass, equals } from 'ramda'
+import { describe } from 'riteway'
+
+import { PREFIX_POET, PREFIX_BARD } from 'Helpers/Bitcoin'
```

Comment:
`PREFIX_POET` is not used anywhere in this file.

Reply:
fixed!